### ISSUE TITLE
Missing download details for nl_kaldi-cgn.tar.gz:kaldi | Home Assistant base station

### ISSUE THREAD

**Issue by TheGroundZero** [rhasspy/rhasspy#168]
In reference to #159 I recently updated the Rhasspy add-on on Home Assistant to v2.5.9

Restarting and checking the logs, I still have the same error

```log
[ERROR:2021-01-18 14:59:13,704] rhasspyserver_hermes: Missing download details for nl_kaldi-cgn.tar.gz:kaldi
Traceback (most recent call last):
  File "/usr/lib/rhasspy/.venv/lib/python3.7/site-packages/quart/app.py", line 1821, in full_dispatch_request
    result = await self.dispatch_request(request_context)
  File "/usr/lib/rhasspy/.venv/lib/python3.7/site-packages/quart/app.py", line 1869, in dispatch_request
    return await handler(**request_.view_args)
  File "/usr/lib/rhasspy/rhasspy-server-hermes/rhasspyserver_hermes/__main__.py", line 628, in api_profiles
    missing_files = rhasspyprofile.get_missing_files(core.profile)
  File "/usr/lib/rhasspy/rhasspy-profile/rhasspyprofile/download.py", line 207, in get_missing_files
    assert file_details, f"Missing download details for {file_value}"
AssertionError: Missing download details for nl_kaldi-cgn.tar.gz:kaldi
[DEBUG:2021-01-18 14:59:13,678] rhasspyprofile.download: speech_to_text.system kaldi kaldi = True
[DEBUG:2021-01-18 14:59:13,677] rhasspyprofile.download: speech_to_text.system pocketsphinx kaldi = False
[INFO:2021-01-18 14:59:13,352] rhasspyserver_hermes: Started
[DEBUG:2021-01-18 14:59:13,352] rhasspyserver_hermes: Subscribed to rhasspy/asr/master/master/audioCaptured
[DEBUG:2021-01-18 14:59:13,351] rhasspyserver_hermes: Subscribed to hermes/nlu/intentNotRecognized
[DEBUG:2021-01-18 14:59:13,351] rhasspyserver_hermes: Subscribed to hermes/audioServer/master/audioSummary
[DEBUG:2021-01-18 14:59:13,350] rhasspyserver_hermes: Subscribed to hermes/asr/textCaptured
[DEBUG:2021-01-18 14:59:13,349] rhasspyserver_hermes: Subscribed to hermes/intent/#
[DEBUG:2021-01-18 14:59:13,349] rhasspyserver_hermes: Subscribed to hermes/hotword/+/detected
[DEBUG:2021-01-18 14:59:13,348] rhasspyserver_hermes: Subscribed to hermes/audioServer/master/audioSummary
[DEBUG:2021-01-18 14:59:13,347] rhasspyserver_hermes: Subscribed to rhasspy/asr/master/master/audioCaptured
[DEBUG:2021-01-18 14:59:13,346] rhasspyserver_hermes: Subscribed to hermes/nlu/intentNotRecognized
[DEBUG:2021-01-18 14:59:13,345] rhasspyserver_hermes: Subscribed to hermes/intent/#
[DEBUG:2021-01-18 14:59:13,345] rhasspyserver_hermes: Subscribed to hermes/asr/textCaptured
[DEBUG:2021-01-18 14:59:13,344] rhasspyserver_hermes: Subscribed to hermes/hotword/+/detected
[DEBUG:2021-01-18 14:59:13,343] rhasspyserver_hermes: Connected to MQTT broker
[DEBUG:2021-01-18 14:59:13,319] rhasspyserver_hermes: Connecting to 192.168.10.105:1883 (retries: 0/10)
[DEBUG:2021-01-18 14:59:13,318] rhasspyserver_hermes: Starting core
[DEBUG:2021-01-18 14:59:13,314] rhasspyprofile.profile: Loading default profile settings from /usr/lib/rhasspy/rhasspy-profile/rhasspyprofile/profiles/defaults.json
[DEBUG:2021-01-18 14:59:13,313] rhasspyprofile.profile: Loading /share/rhasspy/profiles/nl/profile.json
[DEBUG:2021-01-18 14:59:13,312] rhasspyprofile.profile: Loading /usr/lib/rhasspy/rhasspy-profile/rhasspyprofile/profiles/nl/profile.json
[DEBUG:2021-01-18 14:59:12,418] rhasspyserver_hermes: Restarting Rhasspy
```

**Comment by TheGroundZero** [comment]
Training results in the following

```log
[ERROR:2021-01-18 15:40:25,092] rhasspyserver_hermes: Command '['bash', '/usr/lib/rhasspy/.venv/lib/kaldi/egs/wsj/s5/utils/prepare_lang.sh', '/share/rhasspy/profiles/nl/kaldi/model/data/local/dict', '<unk>', '/share/rhasspy/profiles/nl/kaldi/model/data/local/lang', '/share/rhasspy/profiles/nl/kaldi/model/data/lang']' returned non-zero exit status 1.
Traceback (most recent call last):
  File "/usr/lib/rhasspy/.venv/lib/python3.7/site-packages/quart/app.py", line 1821, in full_dispatch_request
    result = await self.dispatch_request(request_context)
  File "/usr/lib/rhasspy/.venv/lib/python3.7/site-packages/quart/app.py", line 1869, in dispatch_request
    return await handler(**request_.view_args)
  File "/usr/lib/rhasspy/rhasspy-server-hermes/rhasspyserver_hermes/__main__.py", line 1321, in api_train
    result = await core.train()
  File "/usr/lib/rhasspy/rhasspy-server-hermes/rhasspyserver_hermes/__init__.py", line 471, in train
    raise TrainingFailedException(reason=asr_response.error)
rhasspyserver_hermes.TrainingFailedException: Command '['bash', '/usr/lib/rhasspy/.venv/lib/kaldi/egs/wsj/s5/utils/prepare_lang.sh', '/share/rhasspy/profiles/nl/kaldi/model/data/local/dict', '<unk>', '/share/rhasspy/profiles/nl/kaldi/model/data/local/lang', '/share/rhasspy/profiles/nl/kaldi/model/data/lang']' returned non-zero exit status 1.
[ERROR:2021-01-18 15:40:25,092] rhasspyserver_hermes: AsrError(error="Command '['bash', '/usr/lib/rhasspy/.venv/lib/kaldi/egs/wsj/s5/utils/prepare_lang.sh', '/share/rhasspy/profiles/nl/kaldi/model/data/local/dict', '<unk>', '/share/rhasspy/profiles/nl/kaldi/model/data/local/lang', '/share/rhasspy/profiles/nl/kaldi/model/data/lang']' returned non-zero exit status 1.", site_id='master', context=None, session_id='08f84966-ac96-4d2d-b069-ad4e238a20df')
[DEBUG:2021-01-18 15:40:25,090] rhasspyserver_hermes: Handling AsrError (topic=hermes/error/asr, id=0c48cb5a-942a-4214-a10c-80cdedce01d4)
[DEBUG:2021-01-18 15:40:21,741] rhasspyserver_hermes: Handling NluTrainSuccess (topic=rhasspy/nlu/master/trainSuccess, id=0c48cb5a-942a-4214-a10c-80cdedce01d4)
[DEBUG:2021-01-18 15:40:21,507] rhasspyserver_hermes: Publishing 6280 bytes(s) to rhasspy/nlu/master/train
[DEBUG:2021-01-18 15:40:21,507] rhasspyserver_hermes: -> NluTrain
[DEBUG:2021-01-18 15:40:21,504] rhasspyserver_hermes: Publishing 6280 bytes(s) to rhasspy/asr/master/train
[DEBUG:2021-01-18 15:40:21,503] rhasspyserver_hermes: -> AsrTrain
[DEBUG:2021-01-18 15:40:21,498] rhasspyserver_hermes: Subscribed to hermes/error/nlu
[DEBUG:2021-01-18 15:40:21,497] rhasspyserver_hermes: Subscribed to rhasspy/nlu/master/trainSuccess
[DEBUG:2021-01-18 15:40:21,497] rhasspyserver_hermes: Subscribed to hermes/error/asr
[DEBUG:2021-01-18 15:40:21,496] rhasspyserver_hermes: Subscribed to rhasspy/asr/master/trainSuccess
[DEBUG:2021-01-18 15:40:21,494] rhasspyserver_hermes: Finished writing /share/rhasspy/profiles/nl/intent_graph.pickle.gz
[DEBUG:2021-01-18 15:40:21,217] rhasspyserver_hermes: Writing /share/rhasspy/profiles/nl/intent_graph.pickle.gz
[DEBUG:2021-01-18 15:40:20,979] rhasspynlu.slots: Running program for slot rhasspy/number,0,100,1: ['/share/rhasspy/profiles/nl/slot_programs/rhasspy/number', '0', '100', '1']
[DEBUG:2021-01-18 15:40:20,964] rhasspynlu.slots: Loading slot hass/colors from /share/rhasspy/profiles/nl/slots/hass/colors
[DEBUG:2021-01-18 15:40:20,719] rhasspynlu.slots: Running program for slot hass/entities,light: ['/share/rhasspy/profiles/nl/slot_programs/hass/entities', 'light']
[DEBUG:2021-01-18 15:40:19,918] rhasspynlu.slots: Running program for slot hass/entities,switch: ['/share/rhasspy/profiles/nl/slot_programs/hass/entities', 'switch']
[DEBUG:2021-01-18 15:40:19,913] rhasspynlu.ini_jsgf: Loaded ini file
[DEBUG:2021-01-18 15:40:19,911] rhasspyserver_hermes: Generating intent graph
[DEBUG:2021-01-18 15:40:19,910] rhasspyserver_hermes: Loading sentences from [PosixPath('/share/rhasspy/profiles/nl/sentences.ini'), PosixPath('/share/rhasspy/profiles/nl/intents/hass.ini')]
[INFO:2021-01-18 15:40:19,909] rhasspyserver_hermes: Starting training
```

**Comment by danielang** [comment]
I have currently the same problem. But I intsalled 2.5.9 on a dedicated lxc container.
Log: 
```
[ERROR:2021-02-02 00:24:39,569] rhasspyserver_hermes: Command '['bash', '/usr/lib/rhasspy/lib/kaldi/egs/wsj/s5/utils/prepare_lang.sh', '/root/.config/rhasspy/profiles/de/kaldi/model/data/local/dict', '<unk>', '/root/.config/rhasspy/profiles/de/kaldi/model/data/local/lang', '/root/.config/rhasspy/profiles/de/kaldi/model/data/lang']' died with <Signals.SIGPIPE: 13>.
Traceback (most recent call last):
  File "/usr/lib/rhasspy/usr/local/lib/python3.7/site-packages/quart/app.py", line 1821, in full_dispatch_request
    result = await self.dispatch_request(request_context)
  File "/usr/lib/rhasspy/usr/local/lib/python3.7/site-packages/quart/app.py", line 1869, in dispatch_request
    return await handler(**request_.view_args)
  File "/usr/lib/rhasspy/rhasspy-server-hermes/rhasspyserver_hermes/__main__.py", line 1321, in api_train
    result = await core.train()
  File "/usr/lib/rhasspy/rhasspy-server-hermes/rhasspyserver_hermes/__init__.py", line 471, in train
    raise TrainingFailedException(reason=asr_response.error)
rhasspyserver_hermes.TrainingFailedException: Command '['bash', '/usr/lib/rhasspy/lib/kaldi/egs/wsj/s5/utils/prepare_lang.sh', '/root/.config/rhasspy/profiles/de/kaldi/model/data/local/dict', '<unk>', '/root/.config/rhasspy/profiles/de/kaldi/model/data/local/lang', '/root/.config/rhasspy/profiles/de/kaldi/model/data/lang']' died with <Signals.SIGPIPE: 13>.
[ERROR:2021-02-02 00:24:39,569] rhasspyserver_hermes: AsrError(error="Command '['bash', '/usr/lib/rhasspy/lib/kaldi/egs/wsj/s5/utils/prepare_lang.sh', '/root/.config/rhasspy/profiles/de/kaldi/model/data/local/dict', '<unk>', '/root/.config/rhasspy/profiles/de/kaldi/model/data/local/lang', '/root/.config/rhasspy/profiles/de/kaldi/model/data/lang']' died with <Signals.SIGPIPE: 13>.", site_id='base', context=None, session_id='c427bf69-0d1d-44aa-bbbb-c175e83a5edf')
[DEBUG:2021-02-02 00:24:39,568] rhasspyserver_hermes: Handling AsrError (topic=hermes/error/asr, id=2da00daf-7a29-46f3-a771-b9745568a56d)
[DEBUG:2021-02-02 00:24:38,732] rhasspyserver_hermes: Handling NluTrainSuccess (topic=rhasspy/nlu/base/trainSuccess, id=2da00daf-7a29-46f3-a771-b9745568a56d)
[DEBUG:2021-02-02 00:24:38,727] rhasspyserver_hermes: Handling NluTrainSuccess (topic=rhasspy/nlu/base/trainSuccess, id=2da00daf-7a29-46f3-a771-b9745568a56d)
[DEBUG:2021-02-02 00:24:38,676] rhasspyserver_hermes: Publishing 293 bytes(s) to rhasspy/nlu/base/train
[DEBUG:2021-02-02 00:24:38,676] rhasspyserver_hermes: -> NluTrain
[DEBUG:2021-02-02 00:24:38,674] rhasspyserver_hermes: Publishing 293 bytes(s) to rhasspy/asr/base/train
[DEBUG:2021-02-02 00:24:38,674] rhasspyserver_hermes: -> AsrTrain
[DEBUG:2021-02-02 00:24:38,672] rhasspyserver_hermes: Finished writing /root/.config/rhasspy/profiles/de/intent_graph.pickle.gz
[DEBUG:2021-02-02 00:24:38,671] rhasspyserver_hermes: Writing /root/.config/rhasspy/profiles/de/intent_graph.pickle.gz
[DEBUG:2021-02-02 00:24:38,669] rhasspynlu.ini_jsgf: Loaded ini file
[DEBUG:2021-02-02 00:24:38,668] rhasspyserver_hermes: Generating intent graph
[DEBUG:2021-02-02 00:24:38,667] rhasspyserver_hermes: Loading sentences from [PosixPath('/root/.config/rhasspy/profiles/de/sentences.ini')]
[INFO:2021-02-02 00:24:38,666] rhasspyserver_hermes: Starting training
```

**Comment by synesthesiam** [comment]
Hopefully fixed in 2.5.10. Please re-open if the problem persists.

**Comment by pablinho1023** [comment]
Hello.
I am having the same issue. Is there a way to fix it? I erased everything from the words tab but still can't train.
